### PR TITLE
Update package exports and peer dependencies

### LINKED ISSUE #113: Tabler UMD is bundled with react build files
Hello. I'm using `react@17.0.1` alongside `@tabler/icons@1.38.1` npm package and its icon components.
I did a performance analyze with [Google PageSpeed Insight](https://developers.google.com/speed/pagespeed/insights/) on my site and an analyze with `source-map-explorer` which is recommended by CRA documentation, [Analyzing the Bundle Size
](https://create-react-app.dev/docs/analyzing-the-bundle-size/). In both cases, I noticed there is a UMD file named `@tabler/icons/icons-react/dist/index.umd.js` in build files.

image

I think it's extravagant to exist a UMD file with `637KB` in build files!
So How can I fix it and avoid bundling a UMD file in build process?

### LINKED ISSUE #115: NPM package complains about React 17+
When installing `@tabler/tabler-icons@1.39.1` I get the following warning:

```js
npm WARN @tabler/icons@1.39.1 requires a peer of react@^16.13.1 but none is installed. You must install peer dependencies yourself.
npm WARN @tabler/icons@1.39.1 requires a peer of react-dom@^16.13.1 but none is installed. You must install peer dependencies yourself.
```

I use React 17+ which works with @tabler/icons just fine. This is going to be problematic since the latest NPM handles peer dependencies differently.

### PR DESCRIPTION
**Pull request by adambrgmn**
This PR updates the main et.al. fields to point to the correct dist files as discussed in #113. It also updates the react peer dependencies to be a bit looser than before, as discussed in #115.

Hope you don't mind that I took the liberty of providing this PR, I know managing a popular OSS library can be cumbersome. I would like you to know that I really appreciate the work and love the quality of the icons!

Closes #113
Closes #115

### CODE REVIEW

**Comment by aulneau** [comment]
Came here to make this same PR. @codecalm anyway you can give this your 👀  and merge it in? Would prefer to not have to fork and publish a different pkg just for these changes.

**Review by aulneau — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -6,8 +6,10 @@
     "url": "git+https://github.com/tabler/tabler-icons.git"
   },
   "main": "./icons-react/dist/index.cjs.js",
+  "exports": "./icons-react/dist/index.esm.js",
   "module": "./icons-react/dist/index.esm.js",
-  "browser": "./icons-react/dist/index.umd.js",
```
[the remark this answers is not in the document]
Comment (adambrgmn) [reply package.json:10]: Oh, thanks! I thought I had verified that, but realized I switched from `umd:main` to `browser` just before making the PR. Can't remember why 🤷‍♂️ I'll fix it.

**Review by FreexD — APPROVED** [review]
LGTM! Thank you for the great work :)

**Comment by Benjozork** [comment]
Thanks for the PR! Our projects depends on Tabler a lot and this was a pain point since we use React 17.0 across the board.

**Comment by theMasix** [comment]
Any package release due to this PR? @FreexD

**Comment by FreexD** [comment]
@theMasix it will be released this week with a bunch of new icons in `v1.40` :)

**Pull request merged**